swarm/network/stream/peer.go

```diff
@@ -154,7 +154,7 @@ func (p *Peer) Deliver(ctx context.Context, chunk storage.Chunk, priority uint8,
 	}
 
 	ctx = context.WithValue(ctx, "stream_send_tag", nil)
-	return p.SendPriority(ctx, msg, priority)
+	return p.Send(ctx, msg)
```

Comment:
why change  it here?
either leave it or change  everywhere by temporarily redefining `sendPriority`

Reply:
I change it so that we can use tracing to debug this problem. `SendPriority` doesn't support traces.

Furthermore I posted measurements of the priority queue in the chat yesterday, which show that messages could stay in the priority queue for tens-of-seconds, which is really bad.

Bottom line, I try to reduce the number of packages that behave weird in order to understand the timeout issue.

Smoke tests have been passing for the past 16 hours consistently with this PR, which means that using `Send` here is just fine at least according to our current test suite.

@zelig do we have any actual test that show the we must use a priority queue for send RPC calls? Generally all our RPC handlers are very fast and the `handleIncoming` loop is really tight, so the priority queue doesn't make much sense to me.

Reply:
good job @nonsense could you try setting the syncer streamer priority from top to mid to understand what was happening